src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,207 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+    
+    public MyLinkedList() {
+        head = null;
+        tail = null;
+        size = 0;
+    }
+    
+    private static class Node<T> {
+        T value;
+        Node<T> next;
+        Node<T> prev;
+        
+        Node(Node<T> prev, T value, Node<T> next) {
+            this.value = value;
+            this.next = next;
+            this.prev = prev;
+        }
+    }
+    
     @Override
     public boolean add(T value) {
-        return false;
+        if (size == 0) {
+            Node<T> newNode = new Node<>(null, value, null);
+            head = newNode;
+            tail = newNode;
+            size++;
+            return true;
+        } else {
+            Node<T> currentNode = head;
+            while (true) {
+                if (currentNode.next == null) {
+                    Node<T> newNode = new Node<>(currentNode, value, null);
+                    currentNode.next = newNode;
+                    tail = newNode;
+                    size++;
+                    return true;
+                }
+                currentNode = currentNode.next;
```

Comment:
you should add the Node at the end of list. You have head and tail, noo reason to iterate